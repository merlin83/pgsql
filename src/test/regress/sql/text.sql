-- *************testing built-in type text  ****************

SELECT 'this is a text string'::text = 'this is a text string'::text AS true;

SELECT 'this is a text string'::text = 'this is a text strin'::text AS false;

CREATE TABLE TEXT_TBL (f1 text);

INSERT INTO TEXT_TBL VALUES ('doh!');
INSERT INTO TEXT_TBL VALUES ('hi de ho neighbor');

SELECT '' AS two, * FROM TEXT_TBL;

