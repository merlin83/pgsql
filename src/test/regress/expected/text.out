QUERY: SELECT 'this is a text string'::text = 'this is a text string'::text AS true;
true
----
t   
(1 row)

QUERY: SELECT 'this is a text string'::text = 'this is a text strin'::text AS false;
false
-----
f    
(1 row)

QUERY: CREATE TABLE TEXT_TBL (f1 text);
QUERY: INSERT INTO TEXT_TBL VALUES ('doh!');
QUERY: INSERT INTO TEXT_TBL VALUES ('hi de ho neighbor');
QUERY: SELECT '' AS two, * FROM TEXT_TBL;
two|f1               
---+-----------------
   |doh!             
   |hi de ho neighbor
(2 rows)

